common/src/main/java/net/apartium/cocoabeans/reflect/ClassUtils.java

```diff
@@ -0,0 +1,25 @@
+package net.apartium.cocoabeans.reflect;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+public class ClassUtils {
+
+    public static Collection<Class<?>> getSuperClassAndInterfaces(Class<?> clazz) {
```

Comment:
Change to a method that gets all method instances from clazz (including inheritance), use reflection cache

Reply:
how is that related to class utils?
We could add reflection cache for classes if that what you want